CallAutomation_Live_Transcription/src/main/java/com/communication/callautomation/ProgramSample.java

```diff
@@ -271,17 +272,27 @@ private void handleIncomingCall(final BinaryData eventData) {
         String callbackUri;
         AnswerCallOptions options;
         String cognitiveServicesUrl;
+        String websocketUrl;
 
         try {
             callbackUri = String.format("%s/%s?callerId=%s",
                     appConfig.getCallBackUri(),
                     UUID.randomUUID(),
                     data.getJSONObject("from").getString("rawId"));
+                    String callbackHostUri = appConfig.getCallBackUri(); // Example: https://3tkj9v9n-8080.inc1.devtunnels.ms/api/callback
+
+                    // Extract the base host and port by removing the path after the host
```

Comment:
why do we need this step? in the app config you will be providing only the host. 
ex: https://3tkj9v9n-8080.inc1.devtunnels.ms/

Reply:
resolved